smartseq2_pipeline/run_multi_samples/scripts/parse_picard.py

```diff
@@ -0,0 +1,77 @@
+from crimson import picard
+import json
+import pandas as pd
+import argparse
+import sys
+from collections import OrderedDict
+import os
+
+def parse_picard_to_table_by_sample(input_metrics_list,output_filename):
+    mets=OrderedDict()
+    with open(input_metrics_list) as f:
+        for line in f:
+            met_fn = line.strip('\n')
+            base = os.path.basename(met_fn)
+            [sid,ftype] = os.path.splitext(base)
```

Comment:
Just wondering, what is `sid`?

Reply:
SRA id, it is run_id or sample id

Reply:
Ok! I think renaming it to `SRA_id` then could help make that clearer!

Reply:
just did